third_party/lua-cjson/lua_cjson.c

```diff
@@ -134,7 +134,7 @@ static const char *char2escape[256] = {
     "\\u0018", "\\u0019", "\\u001a", "\\u001b",
     "\\u001c", "\\u001d", "\\u001e", "\\u001f",
     NULL, NULL, "\\\"", NULL, NULL, NULL, NULL, NULL,
-    NULL, NULL, NULL, NULL, NULL, NULL, NULL, "\\/",
+    NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
```

Comment:
I'll leave here a comment, but, to be honest, I have no strict vision, whether/how it should be resolved.

Should we provide an option for that? If so, should we change a default behaviour? Should the option be global, encoder instance specific or both? (Similar questions appear in the upstream — I summarized it briefly in #6200.)

Or we should go on the long way with some more general behaviour control mechanish, like one I described [here](https://github.com/tarantool/tarantool/discussions/6912)?

Reply:
We decided to use tarantool.compat (#7060), it is now required to merge the patch.